### PR TITLE
Fix class labels [RAPTOR-6493]

### PR DESCRIPTION
**Pull request by gkoundry**
# This repository is public. Do not put here any private DataRobot or customer's data: code, datasets, model artifacts, .etc.
## Summary
For multiclass projects, use class labels from environment or command line option or else get them from the target without sampling the target.

### CODE REVIEW

File: custom_model_runner/datarobot_drum/drum/drum.py [hunk 1]
```diff
@@ -1109,7 +1115,7 @@ def possibly_intuit_order(
             )
             print(e, file=sys.stderr)
             raise DrumCommonException(e)
-        uniq = df[target_col_name].sample(min(1000, len(df)), random_state=1).unique()
+        uniq = df[target_col_name].unique()
```
Comment (4dahalibut) [inline custom_model_runner/datarobot_drum/drum/drum.py:1118]: did we want to sort these?

Comment (gkoundry) [reply custom_model_runner/datarobot_drum/drum/drum.py:1118]: They weren't sorted before this PR.  Is there a reason to?

Comment (4dahalibut) [reply custom_model_runner/datarobot_drum/drum/drum.py:1118]: so the order doesn't change around between runs...

**Review by jmbannon — APPROVED** [review]

File: custom_model_runner/datarobot_drum/drum/drum.py (hunk 1 above)
Comment (gkoundry) [reply custom_model_runner/datarobot_drum/drum/drum.py:1118]: Sort has been added.  Can you approve?

**Review by 4dahalibut — APPROVED** [review]

**Pull request merged**